packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -399,18 +401,15 @@ export class AttestationService {
    * Update the local list of validators based on the current head state.
    */
   private async updateValidators(): Promise<void> {
-    let index = 0;
-    for (const pubkey of this.publicKeys) {
-      // fetch validator details if missing
-      if (!this.validators[index]) {
+    this.validators.forEach(async (v, pk) => {
```

Comment:
Warning! Using an async callback inside a forEach callback is not a good idea. Would recommend using `for..of` loop like before which handles rejections properly